pages.de/linux/dpkg.md

```diff
@@ -0,0 +1,29 @@
+# dpkg
+
+> Debian Paketmanager.
+> Einige Unterbefehle wie `dpkg deb` haben ihre eigene Gebrauchsdokumentation.
```

Comment:
We have this template in the docs: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#de
```suggestion
> Manche Unterbefehle wie `dpkg deb` sind separat dokumentiert.
```